src/components/PercentageTableCell.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { PercentWithValue } from './PercentWithValue';
+import { PercentBar, BarType } from './PercentBar';
+import { numericValue } from 'utils/displayUtils';
+
+export function PercentageTableCell({ type, fraction, total, decimalPlaces }: {
+  type: BarType,
+  fraction: string | number,
+  total: number,
+  decimalPlaces: number
+}) {
+  const val = numericValue(fraction);
+  return (
+    <>
```

Comment:
Whoa, this is cool! I've never seen an empty tag before, but I've wanted something like it!

Reply:
This is a shortcut syntax for [React.Fragment](https://reactjs.org/docs/fragments.html#short-syntax).